readme.usage.md

```diff
@@ -18,7 +18,28 @@ placing it into the topmost parent directory of your repo.
 
 Optionally, you can use another filename and location, and then specify it as a command line argument.
 
-#### 2- sesamconfig
+#### 2- authconfig
+
+In case of using a connector (e.g. Hubspot connector), Sesam client uses `.authconfig` file to authenticate via oauth or 
```

Comment:
I'd just adjust the text slightly - it's when you are working on a connector repository, not really "using a connector". So I'd rephrase it from `In case of using a connector` to `When working on a connector`, or something like that.

Reply:
done